src/PostgREST/Config.hs

```diff
@@ -139,6 +142,7 @@ readOptions = do
           <*> C.key "db-schema"
           <*> (fromMaybe "127.0.0.1" . mfilter (/= "") <$> C.key "server-host")
           <*> (fromMaybe 3000 . join . fmap coerceInt <$> C.key "server-port")
+          <*> C.key "server-unix-socket"
```

Comment:
I've noticed you can pass a config like `server-unix-socket=""` and it works. Do you think that should be allowed?

I've tested that with `echo -e "GET / HTTP/1.0\n" | nc -U ""` and the output seems correct.

Reply:
Further research lead me to this https://stackoverflow.com/questions/29427769/how-to-use-unix-domain-socket-without-creating-a-socket-file.

So perhaps we should allow empty value as a unix socket.

Reply:
I did not know anything about sockets before so no opinion regarding this. I have corrected the example but will leave the empty string as a valid option to allow the usage shown on stack overflow.

Thanks for taking the time to research this.